basic/jms/src/main/resources/META-INF/spring/integration/aggregation.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns:int="http://www.springframework.org/schema/integration"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://www.springframework.org/schema/beans"
+	xmlns:int-jms="http://www.springframework.org/schema/integration/jms"
+	xmlns:int-stream="http://www.springframework.org/schema/integration/stream"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
+			http://www.springframework.org/schema/beans/spring-beans.xsd
+			http://www.springframework.org/schema/context
+			http://www.springframework.org/schema/context/spring-context.xsd
+			http://www.springframework.org/schema/integration
+			http://www.springframework.org/schema/integration/spring-integration.xsd
+			http://www.springframework.org/schema/integration/jms
+			http://www.springframework.org/schema/integration/jms/spring-integration-jms.xsd
+			http://www.springframework.org/schema/integration/stream
+			http://www.springframework.org/schema/integration/stream/spring-integration-stream.xsd">
+
+	<int-stream:stdin-channel-adapter id="stdin" channel="stdinToJmsoutChannel"/>
+
+	<int:channel id="stdinToJmsoutChannel"/>
+	
+	<int:chain input-channel="stdinToJmsoutChannel">
+		<int:header-enricher>
+			<int:correlation-id expression="headers['id']" />
```

Comment:
This doesn't have value for the further process.
We send a message to the `requestTopic` with specific `jms_replyTo` and both `<int-jms:inbound-gateway>` will reply with the proper `jms_correlationId`.
Have you had anything specific thought doing this ?